database_structure/database.py

```diff
@@ -0,0 +1,31 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy_utils import database_exists, create_database
+
+
+def get_engine(user: str, password: str, host: str, port: str, db: str):
+
+    database_url = f"postgresql://{user}:{password}@{host}:{port}/{db}"
+
+    if not database_exists(database_url):
+        print("nie ma DB")
+        create_database(database_url)
+    print("jest DB")
+    engine = create_engine(database_url)
```

Comment:
tutaj robisz engine synchroniczny, lepiej jest robić go jako asynchroniczny, wtedy będzie to w pełni kompatybilne z fastapi